coss/opensource_clubs/models.py

```diff
@@ -175,3 +214,4 @@ class ClubProfile(models.Model):
     """User profile relevant only to the Open Source Clubs."""
     user = models.OneToOneField(UserModel)
     is_captain = models.BooleanField(default=False)
+    is_mentor = models.BooleanField(default=False)
```

Comment:
Do we know if mentors will only be members of a club or is there a chance that we will have external mentors (eg staff)?

Reply:
The latter. Based on the Content Outline mentors are not necessarily members of any Club. I added this here, so ideally they have a profile on the website.

Reply:
This feels a bit problematic, to ask all the mentors to have a profile that is. In any case, let's leave this as is for now.